packages/plugins/webpack.config.ts

```diff
@@ -186,27 +184,24 @@ const config = async (env: any): Promise<Configuration> => {
         issue: {
           include: [{ file: '**/*.{ts,tsx}' }],
         },
-        typescript: { configFile: path.join(process.cwd(), pluginDir, 'tsconfig.json') },
+        typescript: { configFile: path.join(process.cwd(), 'tsconfig.json') },
       }),
       new ESLintPlugin({
         extensions: ['.ts', '.tsx'],
         lintDirtyModulesOnly: Boolean(env.development), // don't lint on start, only lint changed files
       }),
-      ...(env.development ? [new LiveReloadPlugin()] : []),
     ],
 
     resolve: {
       extensions: ['.js', '.jsx', '.ts', '.tsx'],
       unsafeCache: true,
     },
-  };
 
-  if (isWSL()) {
-    baseConfig.watchOptions = {
+    watchOptions: {
       poll: 3000,
-      ignored: /node_modules/,
-    };
-  }
+      ignored: ['**/node_modules', '**/dist', '**/.yarn/.cache'],
```

Comment:
I think we can ignore the entire .yarn directory?

```suggestion
      ignored: ['**/node_modules', '**/dist', '**/.yarn'],
```